atlp-rwanda/falcons-ec-be
# #184637242 Setup API documentation using swagger

What does this PR do?

 * Setup  the API documentation using swagger
 
Description of Task to be completed?

 * Setup  the API documentation using swagger
 
How should this be manually tested?

 *  Clone the project and use the terminal to start the development server by "npm run dev"
 * make sure that you have swagger, swaggerUI and swagger-ui-express installed
 * create folder documentation and 
 * add a file containing swagger configurations in json format

Any background context you want to provide?

none

What are the relevant pivotal tracker/Trello stories?
Tracker: 184637242

Screenshots (if appropriate)
none

Questions:
none
